lib/Api/Api.php

```diff
@@ -121,14 +121,10 @@ public function getLogger()
 
     /**
      * Sets a logger.
-     *
-     * @return $this
      */
-    public function setLogger(LoggerInterface $logger)
+    public function setLogger(LoggerInterface $logger): void
```

Comment:
This is a breaking change and will require a major release of this library.

Reply:
What needs to be done to turn this into an official major release?

Reply:
We should review and test all the open PRs in this repo whether they should be released too once we'll make a major release so we don't have to do it soon again. We should update all the dependencies while doing so. Would you have some spare time to help with that?

Reply:
I can definitely help with updating the dependencies of the project. Are there any specific versions of PHP you guys have in mind?

Reply:
And just to be sure if this really is a breaking change or a bug, the interface that this class is implementing always had [void](https://github.com/php-fig/log/commit/f960c17b71ec7067eaa372f4712219a3c20e5926) (or [null](https://github.com/php-fig/log/commit/af29a61d7191c313b0db21f204a06446e62bf863) if you count the earliest version of the interface) as a return type.